src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -620,7 +620,7 @@ VirtualRepeatController.prototype.containerUpdated = function() {
         this.repeatListExpression,
         angular.bind(this, function(items) {
           if (items && items.length) {
-            this.$$rAF(angular.bind(this, this.readItemSize_));
+            this.readItemSize_();
```

Comment:
Slapping on another comment since we discussed this. 

Basically, $$rAF will return when the browser draws. IE does a smart drawing process where it won't redraw the page unless something changes. Since the DOM element in this case is already drawn, we can pull its dimensions immediately. If we ask to wait for another draw, we don't know when that will fire. Though this is an IE related issue, I can foresee Chrome and Firefox adopting a "redrawOnlyOnChange" features as well, especially since Microsoft is pushing Edge's battery life benefits over Chrome.

The other $$rAF on the container is correct because we are dropping the element into the DOM and we want to wait until it's rendered before it's we can get its size.